core/server/master/src/test/java/alluxio/master/journal/JournalContextTest.java

```diff
@@ -149,11 +151,12 @@ public void pauseBlocksJournalContext() throws Exception {
 
       // after un-pausing, new journal contexts can be created
       lock.close();
+      thread2.run();
       CommonUtils.waitFor("journal context created", journalContextCreated::get,
           WaitForOptions.defaults().setTimeoutMs(5 * Constants.SECOND_MS).setInterval(10));
     } finally {
-      thread.interrupt();
-      thread.join();
+      thread2.interrupt();
+      thread2.join();
```

Comment:
Shall we clean up `thread` here as well just to be safe?

Reply:
done